### ISSUE TITLE
PR changing default RPC port 9944 to 19944

### ISSUE THREAD

**Issue by stewfarq** [nazar-pc/space-acres#122]
https://github.com/nazar-pc/space-acres/blob/16935f62db16fcaae5c16703a9da5a1ec524647f/src/backend/node.rs#L48

This change would allow the native execution of Space Acres (on Windows) to run in parallel with a linux farm (Ubtuntu version) running in WSL.

**Comment by nazar-pc** [comment]
Hm, this is an issue that is the same as what we discussed in https://github.com/nazar-pc/space-acres/issues/121. What I meant there is to submit a pull request, a change to the code itself, rather than creating a new issue. I'll close this one as duplicate.
